quick_migrate.py

```
@@ -0,0 +1,9 @@
+import os
+os.environ['SKIP_BACKGROUND_TASKS'] = 'True'
+os.environ['TESTING'] = 'True'  # To skip even more init
+from app import create_app, db  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)